resources/application-connector/charts/application-registry/README.md

```diff
@@ -12,4 +12,6 @@ This service has the following parameters:
 - **namespace** - The Namespace to which you deploy the Proxy Service. The default Namespace is `kyma-integration`.
 - **requestTimeout** - A time-out for requests sent through the Proxy Service. Provide it in seconds. The default time-out is `10`.
 - **requestLogging** - The flag to enable logging of incoming requests. The default value is `false`.
+- **specRequestTimeout** - A timeout for requests fetching specifications provided by the user. It is provided in seconds. The default timeout is `5`.
```

Comment:
```suggestion
- **specRequestTimeout** - A time-out for requests fetching specifications provided by the user. It is provided in seconds. The default time-out is `5`.
```

Reply:
--> As in [MSG](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/time-out) guidelines that we follow ;)

Reply:
Done